lib/virtual_regional_office/client.rb

```diff
@@ -7,35 +7,54 @@ class Client < Common::Client::Base
     include Common::Client::Concerns::Monitoring
     configuration VirtualRegionalOffice::Configuration
 
-    # Initializes the MAS client.
+    # Initializes the VRO client.
     #
     # @example
     #
-    # VirtualRegionalOffice::Client.new({
-    #  "veteranIcn": "9000682",
-    #  "diagnosticCode": "7101",
-    #  "claimSubmissionId": "1234"
-    # })
+    # VirtualRegionalOffice::Client.new(diagnostic_code: 7101, claim_submission_id: 1234)
     def initialize(params)
-      @veteran_icn = params[:veteran_icn]
-      @diagnostic_code = params[:diagnostic_code]
-      @claim_submission_id = params[:claim_submission_id]
+      @diagnostic_code = params[:diagnostic_code].to_s
+      @claim_submission_id = params[:claim_submission_id].to_s
 
-      raise ArgumentError, 'no veteran_icn passed in for request.' if @veteran_icn.blank?
```

Comment:
Since the two new endpoints don't actually take in `veteran_icn`, I made that parameter a named argument to the one endpoint that does take it in. This unfortunately led to a whole bunch of tiny edits to the tests.